### PR TITLE
Button component

### PR DESCRIPTION
**Pull request by GeetaPogul**
added a button component page.

### CODE REVIEW

File: Button/button.css [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+        /* align-items: center;
+        justify-content: center; */
+        /* padding: 1rem;
+        margin-bottom: 1.9rem;
+        padding-left: 2rem;
+        padding-top: 1rem; */
+        padding:1rem 0rem 1.9rem 2rem;
+        
+    
+}
```
Comment (adidoshi) [inline Button/button.css:93]: Hey, I think we shouldn't mix components css with documentation. Maybe you can move it in a different file. Just a suggestion.

**Review by adidoshi — APPROVED** [review]
Rest, button component code looks fine!👍

File: Badge/badge.css [hunk 2]
```diff
@@ -0,0 +1,146 @@
+@import url(../CSS/navbar.css);
+
+@import url(../CSS/slidebar.css);
+
+img{
```
Comment (msdeshmukh009) [inline Badge/badge.css:5]: Maybe create some class rather than giving styles directly to elements.

File: Badge/badge.css [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+    align-items: center;
+    justify-content: center;
+    padding: 1rem;
+    margin-bottom: 1.5rem;
+    
+}
+
+.sm{
+    width: 40px;
+    height: 40px;
```
Comment (msdeshmukh009) [inline Badge/badge.css:31]: Use of rem will be benificial.

**Review by msdeshmukh009 — APPROVED** [review]

**Pull request merged**